oracle/oracle-how-to/deploy-oke-nodes-using-ubuntu-images.rst

```diff
@@ -0,0 +1,250 @@
+
+- Configured and running OKE cluster on Oracle Cloud.
+
+- Oracle's ``oci`` CLI installed.
+
+- ``kubectl`` installed (Self-Managed only).
+
+
+Download an Ubuntu image
+------------------------
+
+
+To download an image, select a version from the `available releases <#available-releases>`_ and make note of the image path for the image you choose. The image path conforms to the following format:
+
+.. code:: bash
+  
+  <suite>/oke-<version>/<serial>/<image-name>.img
+
+
+Then, from the browser you can paste the image path after the link to begin downloading the image. 
+
+Alternatively, you can use ``curl`` to achieve the same result by using the following command.
+
+.. code:: bash
+   
+  curl -O <available-releases-link>/<image-path>
+
+Upload the Ubuntu image to a bucket
+-----------------------------------
+
+The image must be uploaded to a bucket for image registration later on. Do this in Oracle Cloud by navigating to :guilabel:`Buckets` and selecting your bucket then click :guilabel:`Upload`.
+
+Alternately, the following oci command will upload the object too.
+
+.. code:: bash
+  
+  oci os object put --bucket-name <bucket-name> --file <file-path>
+
+
+Register the Ubuntu image
+-------------------------
+
+Images must be registered to be used by Oracle Cloud services. To register an image in Oracle Cloud, navigate to :guilabel:`Compute` > :guilabel:`Custom Images` and select :guilabel:`Import Image`. 
+
+In the form you must provide your :guilabel:`Compartment`, :guilabel:`Image name`, :guilabel:`Object name` and :guilabel:`Launch mode`. Additionally the fields :guilabel:`Operating System` and :guilabel:`Image type` must be provided and use ``Ubuntu`` and ``QCOW2``, respectively. 
+
+Optionally, this process can be done with the following command:
+
+.. code:: bash
+
+  oci compute image import from-object \
+    --compartment-id <compartment-id> \
+    --display-name <image-name> \
+    --launch-mode <launch-mode> \
+    --image-source-object-name <object-name> \
+    --bucket-name <bucket-name> \
+    --operating-system "Ubuntu" \
+    --operating-system-version <ubuntu-version-number> \
+    --source-image-type QCOW2 \
+    --wait-for-state AVAILABLE
+
+
+Creating OKE Nodes with Ubuntu Images
+-------------------------------------
+
+The following steps on creating nodes assumes you have an existing OKE cluster on Oracle Cloud, but it is not required to have existing nodes. If you don't have an OKE cluster prepared then Oracle's documentation for `creating a cluster`_ is a good place to start.
+
+Creating Managed OKE nodes with Ubuntu
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Managed nodes are node instances whose lifecycle is managed by the OKE service. Since this is a Limited Availability release of Ubuntu images for OKE, you can only create managed nodes through the Oracle Cloud API (``oci`` CLI or SDK). The ability to create managed from nodes from the Oracle Cloud UI will be added at a later.
```

Comment:
```suggestion
Managed nodes are node instances whose lifecycle is managed by the OKE service. Since this is a Limited Availability release of Ubuntu images for OKE, you can only create managed nodes through the Oracle Cloud API (``oci`` CLI or SDK). The ability to create managed nodes from the Oracle Cloud UI will be added later.
```